iroha/iroha.py

```diff
@@ -341,19 +341,26 @@ class IrohaGrpc(object):
     Possible implementation of gRPC transport to Iroha
     """
 
-    def __init__(self, address=None, timeout=None, secure=False):
+    def __init__(self, address=None, timeout=None, secure=False, max_message_length=5 * 1024**3):
```

Comment:
```suggestion
    def __init__(self, address=None, timeout=None, secure=False, *, max_message_length=5 * 1024**3):
```

it would be difficult to remember what "the fourth parameter" means. maybe force it to be a keyword argument?